### PR TITLE
Optimize D for NO_V8_SRC: 5425 -> 5414

### PR DESCRIPTION
**Pull request by 837951602**
Some other similar changings don't make better(at least in my testing), so they don't exist

### CODE REVIEW

File: src/lib/definitions.js [hunk 1]
```diff
@@ -782,6 +782,7 @@ var createParseIntArgDefault;
             ),
             define('(document + [])[12]', HTMLDOCUMENT),
             define('escape(ARRAY_ITERATOR)[30]', NO_OLD_SAFARI_ARRAY_ITERATOR),
+            define('escape((FILTER + [][[]] + [][[]] + true + FILTER)[100])[2]', NO_V8_SRC), // *
```
Comment (837951602) [inline src/lib/definitions.js]: What's the right format to mean a 22-length padding with string alerady ahead?

Comment (fasttime) [inline src/lib/definitions.js]: Nice. There used to be a constants `RP_9_U: 'undefined'`, but it became unused. We could readd it and write the above as

```js
define('escape((FILTER + RP_9_U + RP_9_U + RP_4_N + FILTER)[100])[2]', NO_V8_SRC)

**Comment by fasttime** [comment]
I believe the right feature here is not `NO_V8_SRC` (which is Firefox, Safari, and IE), but a hypothetical `NO_FF_SRC` (for all but Firefox and Safari). I would like to have that feature but it will take some work to do.

**Comment by 837951602** [comment]
That's my fault. I thought there's no \n at the last.

**Comment by fasttime** [comment]
Actually this could still be useful for later. Related: https://github.com/fasttime/JScrewIt/issues/6#issuecomment-279186444

**Pull request closed without merging**